test.js

```diff
@@ -5,3 +5,7 @@ console.log('Hello Universe');
 
 console.log('Hello other stuff');
 
+if(foo === "bar") {
+  console.log('stuff')
```

Comment:
Codacy has a fix for the issue: [Replace `'stuff')` with `"stuff");`](https://app.dev.codacy.org/gh/troubleshoot-codacy/codacy-test-fork-markdown-test/pullRequest?prid=33920)
```suggestion
  console.log("stuff");
```